apps/civicsignalblog/src/lib/data/json/applications.js

```diff
@@ -0,0 +1,12 @@
+const applications = [
+  "CivicSignal",
+  "Explorer",
+  "Topic Mapper",
+  "Source Manager",
+  "Research Blog",
+].map((label) => ({
+  label,
+  value: label.toLowerCase().replace(/\s+/g, "_"),
+}));
```

Comment:
```js
export const MAIN = "main";
export const EXPLORER = "explorer";
export const TOPIC_MAPPER = "topic-mapper";
export const SOURCE_MAPPER = "source-manager";
export const RESEARCH = "research",

export default applications = {
  MAIN: "CivicSignal",
  EXPLORER: "Explorer",
  TOPIC_MAPPER: "Topic Mapper",
  SOURCE_MAPPER: "Source Manager",
  RESEARCH: "Research",
}
```

1. I think the above is much cleaner and allows `import { MAIN } from "applications"` enabling the slug to be reused without being retyped over and over i.e. single place to change if we ever need to.
2. I don't see **Media Data**

Reply:
I have refactored the app name to something very close to your suggestion, since we use the app names in the labels for the select field in the user collection I had to make a minor adjustment to get  the `{value: "", key:""}` array.

The **Media Data** is actually just a page under the MAIN (CivicSignal) its not a separate app that's why I didn't include it in the list